tests/CommunityToolkit.Aspire.Hosting.Bun.Tests/AddBunAppTests.cs

```diff
@@ -120,8 +120,9 @@ public void AddBunNullNameThrows()
     public void AddBunEmptyNameThrows()
     {
         var builder = DistributedApplication.CreateBuilder();
+        var name = "";
 
-        Assert.Throws<ArgumentException>(() => builder.AddBunApp(""));
+        Assert.Throws<ArgumentException>(() => builder.AddBunApp(name));
```

Comment:
These changes are because a new analyzer emits a warning if you pass an empty/whitespace string literal as an argument here. This is just a trick to bypass the analyzer.